css files/gethelpnow-style.css

```diff
@@ -187,35 +192,42 @@ body {
 	color: #ffffff;
 }
 
+/* ===WINDOW=== */
 .window {
 	height: 100%;
 	margin-left: 25%;
 	padding: 1px 16px;
 }
 
-::-webkit-input-placeholder { /* WebKit, Blink, Edge */
+/* ===PLACEHOLDER=== */
+/* WebKit, Blink, Edge */
+::-webkit-input-placeholder { 
     color: #d3d3d3;
     font-size: 15px;
 }
 
-:-moz-placeholder { /* Mozilla Firefox 4 to 18 */
+/* Mozilla Firefox 4 to 18 */
+:-moz-placeholder { 
    color: #d3d3d3;
    font-size: 15px;
    opacity:  1;
 }
 
-::-moz-placeholder { /* Mozilla Firefox 19+ */
+/* Mozilla Firefox 19+ */
+::-moz-placeholder { 
```

Comment:
Kindly remove this block (from line 216 to 221) as it's same a the previous block ( from line 209 to 214).